news/4.35/platform.html

```diff
@@ -97,6 +97,32 @@ <h2>General Updates </h2>
         </p>
       </td>
   </tr>
+  <tr id="edgeBrowserDefault"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+    <td class="title">Edge as Default Browser on Windows</td>
+    <td class="content">
+      <p>
+        Edge replaces Internet Explorer as the default browser instantiated by SWT. In consequence, every SWT- and
```

Comment:
Can we highlight some of the improvements and reasons why we do this?
- IE engine was heavily out-of-date and out of maintenance -> modern engine will re-enable / fix many use-cases
- Dark theme support via `@media (prefers-color-scheme: dark) { ... }` in CSS

Reply:
That absolutely makes sense. I have rephrased the first paragraph to point to reasons/benefits and moved everything else to a second paragraph. I integrated your proposals, but I have to admit that I do not know that much about better capabilities of WebView2 beyond maintenance and performance, so if you think there is something we should add, would be great if you can simply propose it (maybe just make a commit on top of this adding according information), @sratz. Thank you!